package.json

```diff
@@ -4,6 +4,7 @@
   "scripts": {
     "postinstall": "husky install && patch-package && npm run link:angular -w @coveo/atomic-angular-samples",
     "dev:atomic": "concurrently \"npm run -w @coveo/headless dev\" \"npm run -w @coveo/atomic dev\"",
+    "clean": "nx run-many --target=clean",
```

Comment:
While testing, I frequently needed to clean the packages, so I added a clean task here.